docs/source/services/timelock_service/installation.rst

```diff
@@ -79,13 +79,12 @@ Add client(s) to Timelock
 
 9. Configure each client to use Timelock.
    Detailed documentation is :ref:`here <timelock-client-configuration>`.
-   You must remove any ``leader``, ``timestamp``, or ``lock`` blocks; the ``timelock`` block to add looks like this:
+   You must remove any ``leader``, ``timestamp``, or ``lock`` blocks; the ``timelock`` block to add looks like this.
```

Comment:
nit: I preferred the colon at the end; alternatively, replace `this` with `the below`.

Reply:
Yeah, I've reverted to the colon. Agree that it does read (marginally?) better.